src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/InnerAssignmentCheckTest.java

```diff
@@ -58,7 +58,7 @@ public void testMethod() throws Exception {
     }
 
     @Test
-    public void testDemoBug1195047Comment3() throws Exception {
+    public void testInputInnerAssignmentCheckLoopsAndConditionals() throws Exception {
```

Comment:
The test method name doesn't include the word `Input` by convention

Please change : 
testInputInnerAssignmentCheckLoopsAndConditionals() --> `testInnerAssignmentCheckLoopsAndConditionals()`

Update the same for all such occurences.